backend/tests/conferences/factories.py

```diff
@@ -119,6 +119,19 @@ def durations(self, create, extracted, **kwargs):
 
                 self.durations.add(duration)
 
+    @factory.post_generation
+    def audience_levels(self, create, extracted, **kwargs):
+
+        if not create:
+            return
+
+        if extracted:
+            for audience_level in extracted:
+                self.audience_levels.add(
+                    AudienceLevel.objects.get_or_create(name=audience_level)[
```

Comment:
Just need to fix the formatting here

Reply:
Now that we have pre-commit it should fix... no?